src/runtime-rs/crates/runtimes/virt_container/src/sandbox.rs

```diff
@@ -150,6 +154,16 @@ impl Sandbox for VirtSandbox {
             .await
             .context("setup device after start vm")?;
 
+        // get guest hook path
+        // TODO: maybe add a function for toml_config to get config from hypervisor name since there are multiple places do this
+        let hypervisor_name = &self.config.runtime.hypervisor_name;
+        let hypervisor_config = self
+            .config
+            .hypervisor
+            .get(hypervisor_name)
+            .ok_or_else(|| anyhow!("failed to get hypervisor for {}", &hypervisor_name))
+            .context("get hypervisor")?;
```

Comment:
`context` is redundant, because `ok_or_else` should have processed the detailed error message.

Reply:
Thanks @liubin , fixed.